src/viz/barViewer.js

```diff
@@ -81,6 +81,10 @@ var barViewer = function() {
 		var globalSelection = selection.select('.iobio-bar-1').datum( selection.datum() )
 		globalBar(globalSelection, options);
 
+		// Ensure bars don't interfere with brush interaction
+		selection.select('.iobio-bar-1').selectAll('.rect rect')
```

Comment:
Generally I would prefer avoiding inline styles for something like this. It's better to only set the class with d3 then control the styles in a stylesheet somewhere. Since it's just a small fix this is fine but something to keep in mind.